tests/functional/firstrun_sign_in.js

```diff
@@ -54,11 +54,10 @@ define([
         .then(FunctionalHelpers.testIsBrowserNotified(self, 'fxaccounts:can_link_account'))
         .then(FunctionalHelpers.testIsBrowserNotified(self, 'fxaccounts:login'))
 
-        .findByCssSelector('#fxa-settings-header')
-        .end()
-
-        // user should be unable to sign out.
-        .then(FunctionalHelpers.noSuchElement(self, '#signout'))
+        // A lot of settings do not work in an iframe due to CSP. Do not
```

Comment:
Hm does CSP have to do anything with this though?
Are you thinking `X-Frame-Options`? 

AFAIK CSP is not enabled in any way